app/models/user.rb

```diff
@@ -16,6 +16,22 @@ class User < ApplicationRecord
 
   validates :email, format: {with: URI::MailTo::EMAIL_REGEXP}
 
+  enum :team, [
+    :london,
```

Comment:
How would you feel about declaring and storin the enum as strings? I prefer it as wherever you are you know what the value means, rather than having to convert an integer to the right value?

It would looke like this:
```
{
  london: "london",
   ...
}
```

and the column would need to be a string.

Reply:
Yep can do, agree it would be more easily readable. I didn't think to do it because the values would be the same.